docs/using.rst

```diff
@@ -230,46 +254,58 @@ expected output is::
-  is synced to the equivalent path on the guest VM (``/home/john``). For Windows users there is
-  a little caveat since their home directory (for example `C:\Users\john`) must be mapped to a Unix style
+  In the suggested default configuration, your home directory on the host (for example, ``/home/john``)
+  is synced to the equivalent path on the guest VM (``/home/john``). For Windows users, there is
+  a little caveat, their home directory (for example, `C:\Users\john`) must be mapped to a Unix style
   path (``/c/users/john``).
 
-* `NFS <https://www.vagrantup.com/docs/synced-folders/nfs.html>`_: works with Linux/GNU and OS X.
+* `NFS <https://www.vagrantup.com/docs/synced-folders/nfs.html>`_: Works with Linux/GNU and OS X.
 
 There are also some other alternatives, which are, however, not yet properly tested with ADB.
 
 * `SMB <https://www.vagrantup.com/docs/synced-folders/smb.html>`_: For Microsoft Windows.
 
-  * You need to install cifs-utils RPM i.e. ``sudo yum install cifs-utils`` inside ADB for this to work.
+  * You need to install cifs-utils RPM inside ADB, for the SMB synced folder type to work::
+
+    ``sudo yum install cifs-utils``
 
 * `Virtualbox shared folder  <https://www.virtualbox.org/manual/ch04.html#sharedfolders>`_: For Virtualbox users with Virtualbox guest additions.
 
   * At this point of time Virtualbox guest additions do not come pre-installed in the ADB Vagrant box.
-  * For installation details please refer to `Virtualbox documentation <https://www.virtualbox.org/manual/ch04.html>`_.
+  * For installation details, please refer to `Virtualbox documentation <https://www.virtualbox.org/manual/ch04.html>`_.
   * You can also use `vagrant-vbguest <https://github.com/dotless-de/vagrant-vbguest>`_ plugin to install Virtualbox guest additions in ADB Vagrant box.
 
 
-Destroying the Vagrant Box
-==========================
+Some Useful Commands
+====================
+* ``vagrant halt`` - Stop the vagrant box, temporarily:
+
+  You can use ``vagrant halt`` to gracefully stop the vagrant box and continue with
+  your work when you start next with ``vagrant up``. This will not cause any loss
+  of data. It is recommended to stop the vagrant box before you shutdown your machine,
+  to save CPU and RAM consumption. Also, powering off your machine without stopping
+  the vagrant box, could cause errors when you resume using it.
+
+* ``vagrant status`` - Check the Status of the Vagrant box:
 
-**Warning:**
-Doing this will destroy any data you have stored in the Vagrant box. You will
-not be able to restart this instance and will have to create a new one using
-``vagrant up``.
+  Use ``vagrant status`` to check the status of ADB and to check which virtualization
+  provider is being used and the status of the provider.
 
-::
+* ``vagrant destroy`` - Destroy the Vagrant Box:
 
-    vagrant destroy
+  **Warning:**
+  Using``vagrant destroy``, will destroy any data you have stored in the Vagrant
```

Comment:
@Preeticp here vagrant destroy is not rendering in as code block. Space is needed after Using word.